### ISSUE TITLE
Suggestions for additional colors

### ISSUE THREAD

**Issue by fchristant** [yeun/open-color#85]
First of all, fantastic project. It's very difficult to pick shades of attractive hues that have an equal perceived brightness, yet you managed somehow. Extremely valuable.

I know you've commented on some color suggestions already, but I still like to make a plea for a small set of color additions. 

**Brown**
I know you mentioned that you have no plans for it. Justification to perhaps reconsider is that brown is the only missing hue compared to the widely used Material Design colors. I much prefer the consistency of Open Color, and adding brown would make Open Color a feature complete rival in terms of hues, yet with better picked shades. 

**Cold, Neutral, Warm Gray**
The current gray is not neutral, it's slightly cool (blue). I believe gray needs special attention as it is often used as a main UI background, and even the slightest hue change has a far reaching effect on the whole UI. 

My suggestion would be to have a truly neutral gray ramp, a slightly cold one (like the current one), and a slightly warm one. 

Whatever your answer, again, great work!

**Comment by EanKeen** [comment]
I also would love a brown color as well, and curious of the decision made at #60 (with no plans to implement). If there is no way to make brown interoperable with all the other colors (in terms of looking great), then I wouldn't really want to have it as an option.

**Comment by fchristant** [comment]
@EanKeen Maybe this helps:
https://ferdychristant.com/color-for-the-color-challenged-884c7aa04a56

**Comment by yeun** [comment]
Sadly, I still have no plans to make a brown palette. This is because brown is not a hue but a brightness problem unlike other colors. Brown is a color with lower brightness in red. This is the same reason why I did not make khaki or dark navy colors.

But I am open to making gray with different hue. When I first created Open Color, it was common to use cool gray in UI design, so I did not mind other gray. However, to reflect recent trends, I'm considering four gray palettes: 'normal cool gray', 'warm gray', 'neutral gray' and 'very blue gray'.
